gtsfm/common/image.py

```diff
@@ -34,12 +41,51 @@ def width(self) -> int:
         """
         return self.value_array.shape[1]
 
+    def __read_from_exif_image_width(self) -> float:
+        """
+        Compute sensor_width_mm from `ExifImageWidth` tag,
```

Comment:
nit: 1-line docstring should go on line 45 inline, please see the gtsfm style guide [here](https://github.com/borglab/gtsfm/blob/master/CONTRIBUTING.md#python-style). Thanks!

Reply:
Fixed.

Reply:
Marking as resolved, thanks!